mvp-spec/partials/generated-model/get-id-prefs-response.md

```diff
@@ -0,0 +1,314 @@
+# GET /v1/id-prefs response
+
+**Title:** GET /v1/id-prefs response
```

Comment:
Extremly verbose output and inconsistency (e.g a section for a "None" property).

Reply:
I think you're looking at an extremely old version of the code 😄

Reply:
For the record: there were indeed files that remained in an old directory that I forgot to delete.
Good spot, thanks!